brickstrap.sh

```diff
@@ -428,15 +428,71 @@ function brp_preseed_debconf() {
     rm "$(br_rootfs_dir)/tmp/debconfseed.txt"
 }
 
+#
+# Ensure that a 'usable' /etc/shells file exists for the add-shell utility.
```

Comment:
This bug has already been fixed in unstable, so it seems like this workaround is no longer needed.